tests/ui/must_use_pure.stderr

```diff
@@ -0,0 +1,10 @@
+error: this looks like a pure function without `#[must_use]` attribute
+  --> $DIR/must_use_pure.rs:17:1
+   |
+LL | pub fn pure(i: u8) -> u8 {
+   | ^^^^^^^^^^^^^^^^^^^^^^^^ help: add the attribute: `#[must_use] pub fn pure(i: u8) -> u8`
+   |
+   = note: `-D clippy::pure-without-must-use` implied by `-D warnings`
```

Comment:
It would be nice to have more pure function examples that trigger the lint. Does it trigger in pure trait methods, for example?